File: fibonacci_files/fib.cpp
#include <iostream>

using namespace std;

int fib(int n)
{
    if(2 > n)
    {
        return 1;
    }
    else
    {
        return fib(n-1)+fib(n-2);
    }

}

int main()
{
    for(int i=1;i<41;i++){
        cout << i << " . fibonacci: " << fib(i) << endl;
    }
    return 0;
}

File: fibonacci_files/fib.php
<?php
function fib($n) {
    if($n < 2){
        return(1);
    }else{
        return(fib($n-1) + fib($n-2));
    }
}


for ($x = 0; $x < 41; $x++) {
    echo fib($x);
    echo "\n";
} 

?>
